core/spec/lib/spree/core/testing_support/factories/order_factory_spec.rb

```diff
@@ -45,4 +45,10 @@
 
     it_behaves_like 'a working factory'
   end
+
+  describe 'completed order with promotion' do
+    let(:factory) { :completed_order_with_promotion }
+
+    it_behaves_like 'a working factory'
+  end
```

Comment:
I had definitely considered adding more robust tests but I wasn't sure if that's something we wanted to do for our factories. If anyone would like to see more work added to this PR just let me know.